frontend/src/components/AlterSchemaPrepForm/AlterSchemaPrepForm.vue

```diff
@@ -99,8 +101,8 @@ import {
   ProjectId,
   Repository,
   UNKNOWN_ID,
-} from "../../types";
-import { sortDatabaseList } from "../../utils";
+} from "@/types";
+import { allowGhostMigration, isDev, sortDatabaseList } from "@/utils";
 import { cloneDeep } from "lodash-es";
```

Comment:
`lodash-es` is a 3-party package, should be import before local modules.

Reply:
Updated, PTAL